src/controllers/CarsByIDController.js

```diff
@@ -0,0 +1,15 @@
+import BaseController from "./BaseController.js";
+
+
+export default class CarsByIDController extends BaseController{
+    #GET_USER_CARS_PATH = '/api/cars/{id}'
+
+    constructor(request) {
+        super(request)
+    }
+
+    async getUserCars(id){
+        return this._request.get(this.#GET_USER_CARS_PATH.replace('{id}', id))
+    }
+
+}
```

Comment:
you should have only one controller - CarController with all needed methods

Reply:
otherwise it does not make sense to uses classes, you could use regular functions